### PR TITLE
rls: update proto

### PR DESCRIPTION
**Pull request by creamsoup**
the deprecated fields in this import will be removed shortly. update related code

### CODE REVIEW

File: rls/src/main/java/io/grpc/rls/CachingRlsLbClient.java [hunk 1]
```diff
@@ -133,15 +132,9 @@ private CachingRlsLbClient(Builder builder) {
...
-      refCountedChildPolicyWrapperFactory =
-          new RefCountedChildPolicyWrapperFactory(
-              childLbHelperProvider, new BackoffRefreshListener());
-    } else {
-      refCountedChildPolicyWrapperFactory =
-          new RefCountedChildPolicyWrapperFactory(childLbHelperProvider, null);
-    }
+    refCountedChildPolicyWrapperFactory =
+        new RefCountedChildPolicyWrapperFactory(
+            childLbHelperProvider, new BackoffRefreshListener());
```
Comment (dapengzhang0) [inline rls/src/main/java/io/grpc/rls/CachingRlsLbClient.java:137]: The 2nd constructor arg no longer `@Nullable`?

File: rls/src/main/java/io/grpc/rls/CachingRlsLbClient.java [hunk 2]
```diff
@@ -852,13 +845,13 @@ public PickResult pickSubchannel(PickSubchannelArgs args) {
           case READY:
             return childPolicyWrapper.getPicker().pickSubchannel(rlsAppliedArgs);
           case TRANSIENT_FAILURE:
-            return handleError(rlsAppliedArgs, Status.INTERNAL);
+            // fall-through
```
Comment (dapengzhang0) [inline rls/src/main/java/io/grpc/rls/CachingRlsLbClient.java:848]: nit: No need comment. The code explains itself.

File: rls/src/main/java/io/grpc/rls/CachingRlsLbClient.java (hunk 1 above)
Comment (creamsoup) [reply rls/src/main/java/io/grpc/rls/CachingRlsLbClient.java:137]: done.

File: rls/src/main/java/io/grpc/rls/CachingRlsLbClient.java (hunk 2 above)
Comment (creamsoup) [reply rls/src/main/java/io/grpc/rls/CachingRlsLbClient.java]: done.

**Pull request merged**